src/App.js

```diff
@@ -27,13 +27,18 @@ function App() {
     const { data: types, isSuccess: typesSuccess } = useAllTypes();
     const { data: users } = useAllUsers();
 
-    const [loggedInUser, setLoggedInUser] = useState(0);
+    const [loggedInUser, setLoggedInUser] = useState({
+        userID: 0,
```

Comment:
you may want to make a variable to store the default state of the logged in user. That way you can just reset it to that state whenever you log out